.changeset/five-geese-crash.md

```diff
@@ -0,0 +1,9 @@
+---
+'astro': major
+---
+
+Adjust default [View Transition](https://docs.astro.build/en/guides/view-transitions/) animations.
```

Comment:
```suggestion
Change the [View Transition built-in animation](https://docs.astro.build/en/guides/view-transitions/#built-in-animation-directives) options.
```
I think this sounds a little more stand up and take notice. Also links directly to the section.